### ISSUE TITLE
How to install MAgent?

### ISSUE THREAD

**Issue by SHITIANYU-hue** [PKU-AI-Edge/DGN#15]
I clone the MAgent, but I didn't find magent.model

**Comment by zzchuman** [comment]
你好，magent 这个类在哪里啊？

**Comment by SHITIANYU-hue** [comment]
https://github.com/geek-ai/MAgent

maybe this one?🧐

**Comment by sherryzhao0428** [comment]
> I clone the MAgent, but I didn't find magent.model

I have the same problem. I have downloaded the MAgent from the link you provided, I have the same problem, "no module named 'magent.mode'". Did you finally figure it out?

**Comment by SHITIANYU-hue** [comment]
Yes, you could try maybe pip install -e.  Then you need also put the config files into the correct directory

**Comment by jiawweiL** [comment]
hello,I have the following problems，OSError: /home/ljw/文档/DGN/Battle/magent/../../build/libmagent.so: cannot open shared object file: No such file or directory，Do you know what to do? Thank you

**Comment by WayneHsiao0225** [comment]
> I clone the MAgent, but I didn't find magent.model
iI
I have the same problem. Did you solve it? Which  magent package version should  I install? Thank you in advance.

**Comment by WayneHsiao0225** [comment]
> I clone the MAgent, but I didn't find magent.model

I have the same problem. Did you solve it? Which magent package version should I install? Thank you in advance.
@SHITIANYU-hue

**Comment by LLLWQ** [comment]
Hello, I would like to know where to put the clone files? I used Ubuntu 20.0 and anoconda run this project, Which file should be put into the current project?

**Comment by LLLWQ** [comment]
> hello,I have the following problems，OSError: /home/ljw/文档/DGN/Battle/magent/../../build/libmagent.so: cannot open shared object file: No such file or directory，Do you know what to do? Thank you

Hello, I have the same problem. Have you solved it？
